platform/example.anchor-config.yaml

```diff
@@ -0,0 +1,100 @@
+stellar:
+  anchor:
+    # Configure the application from this file after it is loaded in memory
+    # If `config` is in-memory, the yaml file contains all settings for the server.
+    config: in-memory
+    #
+    # Configures how the applications read the
+    #
+    app-config:
+      type: config-spring-property                # Activate [config-spring-property] module
+      settings: app-config                        # The location of the configuration data
+    data-access:
+      type: data-spring-jdbc                      # Activate [config-spring-jdbc] module.
+      settings: data-spring-jdbc-settings         # The location of the configuration data in this file.
+    logging:
+      type: logging-logback
+      settings: logging-logback-settings
+
+#
+# Application settings
+#
+app-config:
+  # general
+  app:
+    stellarNetworkPassPhrase: Test SDF Network ; September 2015
+    hostUrl: http://localhost:8080
+    languages: en
+    horizonURI: https://horizon-testnet.stellar.org
+    assets: assets-test.json
+    jwtSecretKey: secret
+
+  # sep-1
+  sep1:
+    enabled: true
+    stellarFile: sep1/stellar-wks.toml
+
+  # sep-10
+  sep10:
+    enabled: true
+    homeDomain: localhost:8080
+    clientAttributionRequired: false
+    clientAttributionAllowList: lobstr.co,preview.lobstr.co
```

Comment:
In a real case scenario, should we allow all kinds of domains? If so, how to do that?

Reply:
If we allow all, we will use `clientAttributionDenyList`